x-pack/plugins/search_playground/public/components/chat.tsx

```diff
@@ -95,11 +99,17 @@ export const Chat = () => {
       data: buildFormData(formData),
     });
     setIsRegenerating(false);
+
+    usageTracker.click('chat_regenerate_messages');
+  };
+  const handleClearChat = () => {
+    setMessages([]);
+    usageTracker.click('chat_cleared');
```

Comment:
Curious if this will add any value on future product decision?

Reply:
Don't think so, but would be better to have this data in case we need